homeassistant/components/wled/light.py

```diff
@@ -247,6 +262,11 @@ async def async_turn_on(self, **kwargs: Any) -> None:
         if ATTR_RGBW_COLOR in kwargs:
             data[ATTR_COLOR_PRIMARY] = kwargs[ATTR_RGBW_COLOR]
 
+        if ATTR_RGBWW_COLOR in kwargs:
+            data[ATTR_COLOR_PRIMARY] = util.color.color_rgbww_to_rgb(
+                *(kwargs[ATTR_RGBWW_COLOR] + (COLOR_TEMP_K_MIN, COLOR_TEMP_K_MAX))
+            )
```

Comment:
Same, what is the use of adding support for something that actually isn't support but just converted?

Reply:
It allows users to choose the color. Right now the UI is broken. It shows you that you *can* choose a color but when you click on it, nothing happens. With this change you are actually able to change the color and it is also properly displayed back in HA.

Reply:
This is the current behavior:
![wled_125656](https://github.com/user-attachments/assets/a3856c25-0ee3-47d8-95c6-6a5a1656e2fa)

Reply:
This is how it looks with the patch applied. Physically the LEDs match HA. 
![wled_125656_fixed](https://github.com/user-attachments/assets/8d9ca069-3b1c-49f6-9556-9f552d2f0369)

Reply:
I think this is a different issue we should not work around.

This PR adds fake support for RGBWW, which isn't something we should add to Home Assistant.